ibis/expr/format.py

```diff
@@ -162,7 +162,7 @@ def pretty(node):
 
     def mapper(op, _, **kwargs):
         result = fmt(op, **kwargs)
-        if isinstance(op, ops.Relation):
+        if isinstance(op, ops.Relation) and not isinstance(op, ops.JoinTable):
```

Comment:
Since join tables can/should only occur in `JoinChains` it doesn't worth to show them in the repr.

Reply:
Nice, this tightens up the repr quite a bit with long join chains.